pkg/apiserver/statement/col_name.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package statement
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/jinzhu/gorm"
+	"github.com/thoas/go-funk"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/utils"
+)
+
+type fieldSchema struct {
+	DBName  string
+	JSON    string
+	Agg     string
+	Related []string
+}
+
+func getFieldSchema() map[string]fieldSchema {
+	fs := map[string]fieldSchema{}
+
+	utils.ForEachField(Model{}, func(f reflect.StructField) {
```

Comment:
This abstraction / simplify is a baby step. It only reduces specific duplications (5 LOC), but does not provide abstractions (reduce concepts). For example, you still need to learn the Golang's reflect API (`reflect.StructField`) in order to use it.

A good simplification / abstraction reduces both code and concept. Then I found a library https://pkg.go.dev/github.com/oleiade/reflections and you may like it.

Reply:
Inspire me again.
1. Abstraction leak about reflect 
2. No iterator

Reply:
Inspired me again.
1. "Does not reduce concepts". This is about code boundary. My abstraction is like transparent, which is bad.
2. There is no concept of iteration to be abstracted here. Does not represent any real meaning.

Reply:
Love this review more than the lib. lol.